Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM circleci/node:8.9.4 as builder
```

Comment:
The build on Docker hub failed:

```
Building in Docker Cloud's infrastructure...
Cloning into '.'...

KernelVersion: 4.4.0-1060-aws
Components: [{u'Version': u'18.03.1-ee-1-tp5', u'Name': u'Engine', u'Details': {u'KernelVersion': u'4.4.0-1060-aws', u'Os': u'linux', u'BuildTime': u'2018-06-23T07:58:56.000000000+00:00', u'ApiVersion': u'1.37', u'MinAPIVersion': u'1.12', u'GitCommit': u'1b30665', u'Arch': u'amd64', u'Experimental': u'false', u'GoVersion': u'go1.10.2'}}]
Arch: amd64
BuildTime: 2018-06-23T07:58:56.000000000+00:00
ApiVersion: 1.37
Platform: {u'Name': u''}
Version: 18.03.1-ee-1-tp5
MinAPIVersion: 1.12
GitCommit: 1b30665
Os: linux
GoVersion: go1.10.2
Starting build of index.docker.io/teamdigitale/italia-pagopa-proxy:skeleton_with_soap_server...
Step 1/18 : FROM circleci/node:8.9.4 as builder


 ---> 226ab1be62d6

Step 2/18 : RUN sudo apt-get -y install --no-install-recommends libunwind8=1.1-3.2


 ---> Running in 23e61764bd7b

Reading package lists...


Building dependency tree...


Reading state information...


The following NEW packages will be installed:
  libunwind8

0 upgraded, 1 newly installed, 0 to remove and 18 not upgraded.
Need to get 50.8 kB of archives.
After this operation, 186 kB of additional disk space will be used.
Get:1 http://deb.debian.org/debian/ jessie/main libunwind8 amd64 1.1-3.2 [50.8 kB]

[91mdebconf: delaying package configuration, since apt-utils is not installed
[0m
Fetched 50.8 kB in 3s (15.9 kB/s)

Selecting previously unselected package libunwind8.
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 23637 files and directories currently installed.)
Preparing to unpack .../libunwind8_1.1-3.2_amd64.deb ...

Unpacking libunwind8 (1.1-3.2) ...

Setting up libunwind8 (1.1-3.2) ...

Processing triggers for libc-bin (2.19-18+deb8u10) ...

Removing intermediate container 23e61764bd7b

 ---> 62dbed3a3946

Step 3/18 : WORKDIR /usr/src/app


Removing intermediate container fe95be017ea4

 ---> 55f10f52a5f4

Step 4/18 : COPY /src /usr/src/app/src


 ---> 9ce8562c5d97

Step 5/18 : COPY /package.json /usr/src/app/package.json


 ---> b8f559a61284

Step 6/18 : COPY /tsconfig.json /usr/src/app/tsconfig.json


 ---> dd54b15223d7

Step 7/18 : COPY /yarn.lock /usr/src/app/yarn.lock


 ---> 6011ed34b8f7

Step 8/18 : COPY /api_pagopa.yaml /usr/src/app/api_pagopa.yaml


 ---> bce826ed3c40

Step 9/18 : RUN sudo chmod -R 777 /usr/src/app   && yarn install   && yarn build


 ---> Running in 1f8cf9e65782

yarn install v1.3.2

[1/4] Resolving packages...

[2/4] Fetching packages...

[91mwarning Pattern ["request@latest"] is trying to unpack in the same destination "/home/circleci/.cache/yarn/v1/npm-request-2.87.0-32f00235cd08d482b4d0d68db93a829c0ed5756e" as pattern ["request@>=2.9.0","request@^2.55.0","request@^2.83.0"]. This could result in a non deterministic behavior, skipping.
[0m
info fsevents@1.2.4: The platform "linux" is incompatible with this module.

info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.

[3/4] Linking dependencies...

[4/4] Building fresh packages...

Done in 71.70s.

yarn run v1.3.2

$ npm-run-all -s generate:proxy-api-models lint build:tsc

$ rm -f ./src/types/api/* && gen-api-models --api-spec ./api_pagopa.yaml --out-dir ./src/types/api/ --ts-spec-file ./src/api/public_api_pagopa.ts

Writing TS Specs to src/api/public_api_pagopa.ts

CodiceContestoPagamento -> src/types/api//CodiceContestoPagamento.ts

EnteBeneficiario -> src/types/api//EnteBeneficiario.ts

Iban -> src/types/api//Iban.ts

ImportoEuroCents -> src/types/api//ImportoEuroCents.ts

PaymentActivationsGetResponse -> src/types/api//PaymentActivationsGetResponse.ts

PaymentActivationsPostRequest -> src/types/api//PaymentActivationsPostRequest.ts

PaymentActivationsPostResponse -> src/types/api//PaymentActivationsPostResponse.ts

PaymentRequestsGetResponse -> src/types/api//PaymentRequestsGetResponse.ts

RptIdFromString -> src/types/api//RptIdFromString.ts

SpezzoniCausaleVersamento -> src/types/api//SpezzoniCausaleVersamento.ts

SpezzoniCausaleVersamentoItem -> src/types/api//SpezzoniCausaleVersamentoItem.ts

SpezzoneStrutturatoCausaleVersamento -> src/types/api//SpezzoneStrutturatoCausaleVersamento.ts

done

$ tslint --project .

[91mNo valid rules have been specified for TypeScript files
[0m
$ tsc

Done in 25.75s.

Removing intermediate container 1f8cf9e65782

 ---> c490d86da42a

Step 10/18 : FROM node:8.9.4-alpine


 ---> 406f227b21f5

Step 11/18 : LABEL maintainer="https://teamdigitale.governo.it"


 ---> Running in d1b5a5b66006

Removing intermediate container d1b5a5b66006

 ---> b56514c1c1a7

Step 12/18 : WORKDIR /usr/src/app


Removing intermediate container bc5ce8289965

 ---> 6ba789a6a444

Step 13/18 : COPY /package.json /usr/src/app/package.json


 ---> 2ffa45d19c60

Step 14/18 : COPY /public /usr/src/app/public


COPY failed: stat /var/lib/docker/tmp/docker-builder359764167/public: no such file or directory
```